CONTRIBUTING.md

```diff
@@ -1,12 +1,34 @@
-Want to contribute? Great! First, read this page (including the small print at the end).
+# How to Contribute
 
-### Before you contribute
+## Formatting
 
-Before you start working on a larger contribution, you should get in touch
-with us first. Use the issue tracker to explain your idea so we can help and
-possibly guide you.
+Starlark files should be formatted by buildifier.
 
-### Code reviews and other contributions.
-**All submissions, including submissions by project members, require review.**
-Please follow the instructions in [the contributors documentation](https://bazel.build/contributing.html).
+`buildifier --mode fix -lint fix -r .`
```

Comment:
you decided not to reccomend `pre-commit install` like the other repos?

Reply:
That seems to bring in other requirements about how commit messages are formed. Id like to review them properly before copying them into this repo.